adxl345/adxl345.go

```diff
@@ -0,0 +1,230 @@
+package adxl345
+
+import (
+	"encoding/binary"
+	"fmt"
+	"periph.io/x/conn/v3/physic"
+	"periph.io/x/conn/v3/spi"
+)
+
+type Sensitivity byte
+
+const (
+	DeviceID = 0x00 // Device ID, expected to be 0xE5 when using ADXL345
```

Comment:
Please do not export these unless they are meant to be used by the user.

Reply:
It is a Register it is normal to keep it public for anyone that wants to read it.